README.md

```diff
@@ -1,20 +1,93 @@
+Hello everyone and welcome to our club repository! This is a generalized Git Workflow to help gain the skills to collaborate on open source projects. Here you can follow the instructions below to learn how to make a copy of a project from a public repository. Then create changes in your copy on GitHub or download it to your computer to make changes. Last, submit those changes back to the original repository for approval. 
```

Comment:
These are big blocks of text that I don't think anyone will read: can we break these up?

<bolded>Hello...welcome...repository!
<new paragraph, space> This is a generalized....projects.
<white space>
<ordered list>
1.) Follow the instructions below
2.)Create your changes either in the Github UI or locally
3.)Submit changes in a PR to the original repo